internal/util/projutil/project_util.go

```diff
@@ -16,13 +16,15 @@ package projutil
 
 import (
 	"fmt"
+	"github.com/ghodss/yaml"
```

Comment:
Imports order

You should be using `sigs.k8s.io/yaml` which is a k8s-mainatained fork of this library.